src/webgl/p5.Geometry.js

```diff
@@ -143,6 +143,94 @@ p5.Geometry = class Geometry {
     return this;
   }
   /**
+ * Flips the U texture coordinates of the model.
+ * @method flipU
+ * @for p5.Geometry
+ *
+ * @returns {p5.Geometry}
+ *
+ * @example
+ * <div>
+ * <code>
+ * let modelU;
+ *
+ * function preload() {
+ *   modelU = loadModel('path/to/your/model.obj');
```

Comment:
oh wait it's probably because this isn't a "real" model. Maybe try borrowing a model path from e.g. the `loadModel` examples and reusing that here?